tests/behat/actions.feature

```diff
@@ -79,6 +79,50 @@ Feature: Check if all the different type of actions of the mass actions block wo
     And I should see "Test Activity4 (copy)" in the "Topic 4" "section"
     And I should see "Test Activity5 (copy)" in the "Topic 4" "section"
 
+  @javascript
```

Comment:
again, this test would need tweaked if the changes to block_massaction.php are adopted.

Reply:
no need for the test change